universal-application-tool-0.0.1/app/controllers/admin/QuestionController.java

```diff
@@ -201,9 +203,7 @@ public Result update(Request request, Long id, String questionType) {
               .setLifecycleStage(LifecycleStage.DRAFT)
               .build();
     } catch (UnsupportedQuestionTypeException e) {
-      throw new RuntimeException(
-          "Failed while trying to update a question that was already created for question type "
```

Comment:
suggestion (non-blocking): Can we move this message into a comment, so we still have info for when this case happens?

Reply:
Good idea